diff --git a/dist/index.js b/dist/index.js
index 5038836c1407e819e83df75195bf320cc1891203..91de28a330aa72f90ca185af5001481a90cf8561 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -999,6 +999,21 @@ var OpenAIChatLanguageModel = class {
               return;
             }
             const delta = choice.delta;
+            const reasoningContent = delta.reasoning_content;
+            if (reasoningContent) {
+              if (!isActiveReasoning) {
+                controller.enqueue({
+                  type: 'reasoning-start',
+                  id: 'reasoning-0',
+                });
+                isActiveReasoning = true;
+              }
+              controller.enqueue({
+                type: 'reasoning-delta',
+                id: 'reasoning-0',
+                delta: reasoningContent,
+              });
+            }
             if (delta.content != null) {
               if (!isActiveText) {
                 controller.enqueue({ type: "text-start", id: "0" });
@@ -1111,6 +1126,9 @@ var OpenAIChatLanguageModel = class {
             }
           },
           flush(controller) {
+            if (isActiveReasoning) {
+              controller.enqueue({ type: 'reasoning-end', id: 'reasoning-0' });
+            } 
             if (isActiveText) {
               controller.enqueue({ type: "text-end", id: "0" });
             }
diff --git a/dist/index.mjs b/dist/index.mjs
index 2a58125b0f80f9b121c7f1a615737c5a266e822b..0aa2adbc87030e649355eb4dca25a8d0bca1e25e 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -990,6 +990,21 @@ var OpenAIChatLanguageModel = class {
               return;
             }
             const delta = choice.delta;
+            const reasoningContent = delta.reasoning_content;
+            if (reasoningContent) {
+              if (!isActiveReasoning) {
+                controller.enqueue({
+                  type: 'reasoning-start',
+                  id: 'reasoning-0',
+                });
+                isActiveReasoning = true;
+              }
+              controller.enqueue({
+                type: 'reasoning-delta',
+                id: 'reasoning-0',
+                delta: reasoningContent,
+              });
+            }
             if (delta.content != null) {
               if (!isActiveText) {
                 controller.enqueue({ type: "text-start", id: "0" });
@@ -1102,6 +1117,9 @@ var OpenAIChatLanguageModel = class {
             }
           },
           flush(controller) {
+            if (isActiveReasoning) {
+              controller.enqueue({ type: 'reasoning-end', id: 'reasoning-0' });
+            }
             if (isActiveText) {
               controller.enqueue({ type: "text-end", id: "0" });
             }
