diff --git a/dist/index.js b/dist/index.js
index f4c12ee1a8b91abd364e47a32c4cb575532683f9..b9ab94ac5caa676d3c6bc0b445b41cc60f32e5da 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -3271,19 +3271,23 @@ var OpenAIResponsesLanguageModel = class {
                 });
               } else if (isResponseOutputItemDoneReasoningChunk(value)) {
                 const activeReasoningPart = activeReasoning[value.item.id];
-                for (const summaryIndex of activeReasoningPart.summaryParts) {
-                  controller.enqueue({
-                    type: "reasoning-end",
-                    id: `${value.item.id}:${summaryIndex}`,
-                    providerMetadata: {
-                      openai: {
-                        itemId: value.item.id,
-                        reasoningEncryptedContent: (_d = value.item.encrypted_content) != null ? _d : null
+                if ((activeReasoningPart == null ? void 0 : activeReasoningPart.summaryParts) != null) {
+                  for (const summaryIndex of activeReasoningPart.summaryParts) {
+                    controller.enqueue({
+                      type: "reasoning-end",
+                      id: `${value.item.id}:${summaryIndex}`,
+                      providerMetadata: {
+                        openai: {
+                          itemId: value.item.id,
+                          reasoningEncryptedContent: (_d = value.item.encrypted_content) != null ? _d : null
+                        }
                       }
-                    }
-                  });
+                    });
+                  }
+                }
+                if (activeReasoningPart != null) {
+                  delete activeReasoning[value.item.id];
                 }
-                delete activeReasoning[value.item.id];
               }
             } else if (isResponseFunctionCallArgumentsDeltaChunk(value)) {
               const toolCall = ongoingToolCalls[value.output_index];
diff --git a/dist/index.mjs b/dist/index.mjs
index 36d7e71f3bfe096e243a716a87e3c8241803d1f7..1fbecc7b227efe60562e8a9fc2c5907dc0f4872c 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -3295,19 +3295,23 @@ var OpenAIResponsesLanguageModel = class {
                 });
               } else if (isResponseOutputItemDoneReasoningChunk(value)) {
                 const activeReasoningPart = activeReasoning[value.item.id];
-                for (const summaryIndex of activeReasoningPart.summaryParts) {
-                  controller.enqueue({
-                    type: "reasoning-end",
-                    id: `${value.item.id}:${summaryIndex}`,
-                    providerMetadata: {
-                      openai: {
-                        itemId: value.item.id,
-                        reasoningEncryptedContent: (_d = value.item.encrypted_content) != null ? _d : null
+                if ((activeReasoningPart == null ? void 0 : activeReasoningPart.summaryParts) != null) {
+                  for (const summaryIndex of activeReasoningPart.summaryParts) {
+                    controller.enqueue({
+                      type: "reasoning-end",
+                      id: `${value.item.id}:${summaryIndex}`,
+                      providerMetadata: {
+                        openai: {
+                          itemId: value.item.id,
+                          reasoningEncryptedContent: (_d = value.item.encrypted_content) != null ? _d : null
+                        }
                       }
-                    }
-                  });
+                    });
+                  }
+                }
+                if (activeReasoningPart != null) {
+                  delete activeReasoning[value.item.id];
                 }
-                delete activeReasoning[value.item.id];
               }
             } else if (isResponseFunctionCallArgumentsDeltaChunk(value)) {
               const toolCall = ongoingToolCalls[value.output_index];
