diff --git a/dist/index.cjs b/dist/index.cjs
index e96c7203f83b753dd950fd23d4ac5051a337ecc6..bc8b4723a16f72715bd1f83b63dfb177edf73318 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -468,6 +468,19 @@ var DragHandlePlugin = ({
             }
             return false;
           },
+          scroll(view) {
+            if (!element || locked) {
+              return false;
+            }
+            if (view.hasFocus()) {
+              hideHandle();
+              currentNode = null;
+              currentNodePos = -1;
+              onNodeChange == null ? void 0 : onNodeChange({ editor, node: null, pos: -1 });
+              return false;
+            }
+            return false;
+          },
           mouseleave(_view, e) {
             if (locked) {
               return false;
diff --git a/dist/index.js b/dist/index.js
index 309bbba9f2dcdfe1f9cb9e98c1ad9cae6258d253..f66be7dfbf4d5185992445246e43e710d9797260 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -442,6 +442,19 @@ var DragHandlePlugin = ({
             }
             return false;
           },
+          scroll(view) {
+            if (!element || locked) {
+              return false;
+            }
+            if (view.hasFocus()) {
+              hideHandle();
+              currentNode = null;
+              currentNodePos = -1;
+              onNodeChange == null ? void 0 : onNodeChange({ editor, node: null, pos: -1 });
+              return false;
+            }
+            return false;
+          },
           mouseleave(_view, e) {
             if (locked) {
               return false;
