diff --git a/dist/index.cjs b/dist/index.cjs
index 506aa37711fdb8452c68c4e1364b769793e56290..a69f9cc11066f5cf224599cb7b01c7ab6d465bb1 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -454,6 +454,19 @@ var DragHandlePlugin = ({
             }
             return false;
           },
+          scroll(view) {
+            if (!element || locked) {
+              return false;
+            }
+            if (view.hasFocus()) {
+              hideHandle();
+              currentNode = null;
+              currentNodePos = -1;
+              onNodeChange == null ? void 0 : onNodeChange({ editor, node: null, pos: -1 });
+              return false;
+            }
+            return false;
+          },
           mouseleave(_view, e) {
             if (locked) {
               return false;
diff --git a/dist/index.js b/dist/index.js
index ad58ef1637a6e5544733f4002cd0cfcc8e43022a..ce03e2e2882e8d1828726dcb3de31e9cbeb83374 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -428,6 +428,19 @@ var DragHandlePlugin = ({
             }
             return false;
           },
+          scroll(view) {
+            if (!element || locked) {
+              return false;
+            }
+            if (view.hasFocus()) {
+              hideHandle();
+              currentNode = null;
+              currentNodePos = -1;
+              onNodeChange == null ? void 0 : onNodeChange({ editor, node: null, pos: -1 });
+              return false;
+            }
+            return false;
+          },
           mouseleave(_view, e) {
             if (locked) {
               return false;
