diff --git a/lib/jsdom/living/xhr/XMLHttpRequest-impl.js b/lib/jsdom/living/xhr/XMLHttpRequest-impl.js
index 04f033dade5e471387200e58fd62d03ef903dd62..e0b5e512eeee42a07ebe2c0fd71627f2c6a3860b 100644
--- a/lib/jsdom/living/xhr/XMLHttpRequest-impl.js
+++ b/lib/jsdom/living/xhr/XMLHttpRequest-impl.js
@@ -28,7 +28,7 @@ const { serializeEntryList, chunksToBuffer } = require("./multipart-form-data.js
 
 const { READY_STATES } = xhrUtils;
 
-const syncWorkerFile = require.resolve ? require.resolve("./xhr-sync-worker.js") : null;
+const syncWorkerFile = require.resolve ? require.resolve('jsdom/lib/jsdom/living/xhr/xhr-sync-worker.js') : null;
 
 const tokenRegexp = /^[!#$%&'*+\-.^_`|~0-9A-Za-z]+$/;
 const fieldValueRegexp = /^[ \t]*(?:[\x21-\x7E\x80-\xFF](?:[ \t][\x21-\x7E\x80-\xFF])?)*[ \t]*$/;
