diff --git a/dist/index.js b/dist/index.js
index ba480048f147f1620b70c6fcfa5357ade4599beb..3d05ae3a2dfc8fdb1b745a2d206b062dd68630e1 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -536,7 +536,7 @@ function convertToGoogleGenerativeAIMessages(prompt, options) {
 
 // src/get-model-path.ts
 function getModelPath(modelId) {
-  return modelId.includes("/") ? modelId : `models/${modelId}`;
+  return modelId.includes("models/") ? modelId : `models/${modelId}`;
 }
 
 // src/google-generative-ai-options.ts
diff --git a/dist/index.mjs b/dist/index.mjs
index 9e7d7585047db2796a4a783644d78016ae27a520..c40fb884482a40d5342bc2cfe6b766e2718aea91 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -542,7 +542,7 @@ function convertToGoogleGenerativeAIMessages(prompt, options) {
 
 // src/get-model-path.ts
 function getModelPath(modelId) {
-  return modelId.includes("/") ? modelId : `models/${modelId}`;
+  return modelId.includes("models/") ? modelId : `models/${modelId}`;
 }
 
 // src/google-generative-ai-options.ts
@@ -2404,8 +2404,7 @@ function createGoogleGenerativeAI(options = {}) {
 }
 var google = createGoogleGenerativeAI();
 export {
-  VERSION,
   createGoogleGenerativeAI,
-  google
+  google, VERSION
 };
 //# sourceMappingURL=index.mjs.map
