diff --git a/dist/index.js b/dist/index.js
index 8ba8ec4fe78573068683801b75b92cdbe7132b01..cea5346f36296e38ac4fbdf7cf46b330c26fa4f9 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -536,7 +536,7 @@ function convertToGoogleGenerativeAIMessages(prompt, options) {
 
 // src/get-model-path.ts
 function getModelPath(modelId) {
-  return modelId.includes("/") ? modelId : `models/${modelId}`;
+  return modelId.includes("models/") ? modelId : `models/${modelId}`;
 }
 
 // src/google-generative-ai-options.ts
diff --git a/dist/index.mjs b/dist/index.mjs
index 0e96cc445379e63fb3c08a3a6238f1c55878d8b5..7e76a0ed624844b5aa8cfc25b61b901572b48a83 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -542,7 +542,7 @@ function convertToGoogleGenerativeAIMessages(prompt, options) {
 
 // src/get-model-path.ts
 function getModelPath(modelId) {
-  return modelId.includes("/") ? modelId : `models/${modelId}`;
+  return modelId.includes("models/") ? modelId : `models/${modelId}`;
 }
 
 // src/google-generative-ai-options.ts
@@ -1998,8 +1998,7 @@ function createGoogleGenerativeAI(options = {}) {
 }
 var google = createGoogleGenerativeAI();
 export {
-  VERSION,
   createGoogleGenerativeAI,
-  google
+  google, VERSION
 };
 //# sourceMappingURL=index.mjs.map
\ No newline at end of file
