diff --git a/es/index.js b/es/index.js
index c5d5c57488d414e3078f19a235e4db7afebeda80..7892057dab4fbd31ad840f8149978d52f9de1d37 100644
--- a/es/index.js
+++ b/es/index.js
@@ -104680,7 +104680,7 @@ class OutboundSender {
                 const id = yield this.sendOneWithFallback({
                     to, idType,
                     msgType: 'post', content: post,
-                    replyTo: i === 0 ? opts.replyTo : undefined,
+                    replyTo: opts.replyTo,
                     replyInThread: opts.replyInThread,
                 });
                 ids.push(id);
diff --git a/lib/index.js b/lib/index.js
index 0728b1d97ddac688b17642924e359c152dd52fd5..e20f32cd2e27f4f0a7a11d69bf531bf78ef05d72 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -104698,7 +104698,7 @@ class OutboundSender {
                 const id = yield this.sendOneWithFallback({
                     to, idType,
                     msgType: 'post', content: post,
-                    replyTo: i === 0 ? opts.replyTo : undefined,
+                    replyTo: opts.replyTo,
                     replyInThread: opts.replyInThread,
                 });
                 ids.push(id);
